src/components/Lesson.js

```diff
@@ -52,6 +68,7 @@ const Lesson = ({ route, navigation, dispatch, completedGrammars }) => {
       onPressHandler={() => completeBtn()}
     >
       <BodyText>{grammar.description}</BodyText>
+      {showCommonness()}
```

Comment:
Great job! Because this Lesson component is getting a little large, what do you think about moving this `showCommonness` into a new component?